content/doc/addons/elastic/_index.md

```diff
@@ -135,3 +135,28 @@ Backups can be managed under the *Backup* tab of the elastic add-on. You can res
 {{< callout type="warning" >}}
 If you are using Elasticsearch 6, backups are not deleted automatically, you will need to clean them up from time to time.
 {{< /callout >}}
+
+## Create index on Elasticsearch
+
+When creating an Elasticsearch add-on, a single node is set-up. Be default we apply an index template `cc_singlenode_template` that force the amount of data replication to zero. This template is only available for data streams.
+
+To add a new index by API, or when using a tool that requires native Elasticsearch Index, a custom index template is required. This template can be created or by using Kibana or by using the Elasticsearch API.
+
+To do it with the Elasticsearch API, exec in cURL: 
```

Comment:
```suggestion
When creating an Elasticsearch add-on, a single node is setup. By default we apply `cc_singlenode_template` that forces the amount of data replication to zero. This index template is only available for data streams.

If you use a tool that requires native Elasticsearch index or add a new one through the API, a custom template is required. It can be created using Kibana or the Elasticsearch API. For the latter, you can use a cURL request such as:
```